packages/dds/tree/src/util/brandedMap.ts

```diff
@@ -11,10 +11,12 @@ import { getOrCreate } from "./utils.js";
 /**
  * Key in a {@link BrandedMapSubset}.
  * @remarks
+ * `TContent` is used as part of the brand to strongly associate the key type with the type of value it looks up.
```

Comment:
```suggestion
 * `TContent` is used as part of the brand to strongly associate the key type with the type of the value it looks up.
```

Reply:
Since the same BrandedKey can be used in many branded maps, it can be used to look up many values (of the same type). Thus I think adding "the" here doesn't help: there is not a 1:1 relation to values and keys, only between keys and value types, and we already use "the" there.

Reply:
"the type of the values it looks up" would be a possible alternative (note "values" plural there). Maybe that's better?

Reply:
I like that